app/va/va_profile/va_profile_client.py

```diff
@@ -172,7 +172,7 @@ def _make_request(self, url: str, va_profile_id: str, bio_type: str = None):
             response.raise_for_status()
 
         except requests.HTTPError as e:
-            self.logger.exception(e)
+            self.logger.warning('Exception raised making request to VA Profile for VA Profile ID: %s', va_profile_id)
```

Comment:
I would explicitly state "HTTPError", as you do below for push notifications, rather than "Exception".

Reply:
Changed.